src/main/java/com/project/bookstore/dto/user/CreateUserRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.project.bookstore.dto.user;
+
+import com.project.bookstore.validation.Email;
+import jakarta.validation.constraints.NotBlank;
+import org.hibernate.validator.constraints.Length;
+
+public record CreateUserRequestDto(
+        @Email String email,
+        @NotBlank @Length(min = 8) String password,
+        @NotBlank @Length(min = 8) String repeatPassword,

```

Comment:
Not Implemented: Add FieldMatch annotation for checking that password and repeatPassword field in the UserRegistrationRequestDto are the same